draft-meynell-panrg-scion-research-questions.md

```diff
@@ -200,6 +200,26 @@ This draft assumes the reader is familiar with some of the fundamental concepts
 * One AS belongs to many ISDs?
 
 
+## CORE Beaconing Scalability
+
+The idea behind beaconing is to discover all possible paths (loop free and with a fixed maximu length) between two CORE AS.
+Every AS forwards any received becons to all neighbor CORE ASes unless this would cause a loop or exceed the fixed maximum length.
+Implemented naively, the number of paths (and beacons) grows exponentially with the network size.
+This is currently mitigated primarily (and efficiently) by forwarding only the 5 "best PCB" to neighbours.
+
+This "best PCB" practive has several implications:
+
+* It limits the number of available paths for endhosts such that only 5 path may be available to a given destination. This is especially relevant for
+  * Multipathing: you may not be able to use more than five path and no great selection of pats on offer .
+  * Unusual path policies, such as geofencing, may not be fulfillable by the limited number of paths on offer.
+* It causes a small number of path to carry all traffic to a given destination, see {{link-load-balancing}}.
+* It opens the possibility of a DoS attack where a small number of "wormholes", see {{I-D.scion-cp}}, may be selected as "best" and have all traffic
```

Comment:
Are you referring to the `Wormhole Attack` we describe in the CP draft? 
Why is this related to core beaconing scalability?